generator/src/main/java/org/apache/kafka/message/ApiMessageTypeGenerator.java

```diff
@@ -408,6 +410,23 @@ private void generateListenerTypesEnum() {
         buffer.printf("}%n");
     }
 
+    private void generateHighestStableVersion() {
+        buffer.printf("public short highestStableVersion(boolean enableUnstableLastVersion) {%n");
```

Comment:
nit: It's a little odd to call this `highestStableVersion` and also accept a flag to enable unstable versions. Maybe we could replace the implementation of `highestSupportedVersion` to accept the flag.

Or alternatively, the helper can be `highestStableVersion()` and we can drop the flag. Then we can check the flag in the code and use either `highestStableVersion` or `highestSupportedVersion`